plugins/BEdita/Core/src/I18n/MessagesFileLoader.php

```diff
@@ -45,7 +45,7 @@ public function plugins(): array
     /**
      * {@inheritDoc}
      *
-     * @param string[] $plugins Additional plugins to look up in for translations.
+     * @param string[] $name Additional plugins to look up in for translations.
```

Comment:
complete signature is `__construct(string $name, string $locale, string $extension = 'po', array $plugins = [])`.

The additional plugins to look up are `array $plugins`. I guess this change is wrong.

`@inheritDoc` provides the inheritance of the basic signature php doc, `plugins` is the extra parameter. Right?